src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] Please make sure that all of the components from the application are class-based components as required from Microverse. This includes the `App` component and the `CalculatorLayout` component you have created. 

I know this doesn't much sense right now, as components with no state should be functional components, but the objective here it's to practice and understand React on a deeper level.

Reply:
Thank you again. I changed all the components to class-based components.